src/control/security/config.go

```diff
@@ -49,6 +50,10 @@ type TransportConfig struct {
 	CertificateConfig `yaml:",inline"`
 }
 
+func (tc *TransportConfig) String() string {
+	return fmt.Sprintf("allow insecure: %v", tc.AllowInsecure)
+}
+
```

Comment:
This seems unrelated.

Reply:
this is so we can diff a string representation of a struct with transport config pointers